inductiva/utils/files.py

```diff
@@ -100,3 +104,34 @@ def get_alphanum_key(file):
     files = sorted(files, key=get_alphanum_key)
 
     return files
+
+
+def download_from_url(url: str,
+                      local_file_path: str,
+                      save_dir: Optional[str] = None) -> str:
+    """Download a file from an URL.
+
+    Args:
+        url: The URL to download the file from.
+        local_file_path: Name attributed to the file to download.
+        save_dir: The directory to save the file to.
```

Comment:
The docstring does not explain what happens if the default (None) is provided. I am assuming it saves to the current working directory?

Reply:
Also, this is the only extra argument, compared to the urlretrieve underlying method, so it is the only reason why this method was created. One needs to say that if the save_dir does not exist, this method will try to create it. It's also a bit unclear what happens if it is not possible to create that save_dir, I think it will throw an exception, right?

It's a little strange that we are catching an exception if there is an urlib download problem, but we are throwing the one related to creating subfolders... No?

Reply:
Yes, it will throw an exception if it cannot create - done with mkdir.

Given that it will throw an exception first if it does not create it, then it will be fine here.